### PR TITLE
List threads

### PR DESCRIPTION
**Pull request by PrathamKumar14**
List threads that I have posted on Twitter in November

### CODE REVIEW

File: guides/data.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "source": "image-apis"
+  },
+  {
+    "source": "AI-apis"
```
Comment (ghoshnirmalya) [inline guides/data.json]: The source should always be in lower case.

```suggestion
    "source": "ai-apis"
```

File: guides/data.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+  },
+  {
+    "source": "NLP-apis"
```
Comment (ghoshnirmalya) [inline guides/data.json]:
```suggestion
    "source": "nlp-apis"
```

File: guides/posts/AI-apis/post.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Top 5 Artificial Intelligence APIs"
+description: "Top 5 Artificial Intelligence APIs to build excellent full-stack applications."
+slug: "top-artificial-intelligence-apis"
```
Comment (ghoshnirmalya) [inline guides/posts/AI-apis/post.md]: The [source](https://github.com/RapidAPI/DevRel-Stack-Data/pull/79/files#diff-013dbcae57c03571c8d68fc66f69a6a154bdfd5e2bee0a7ef91c7d1da11821eaR6) and the slug should be the same.

File: guides/posts/NLP-apis/post.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
+title: "Top 5 Natural Language Processing APIs"
+description: "Top 5 Natural Language Processing APIs that can help you build excellent web projects."
+slug: "top-natural-language-processing-apis"
```
Comment (ghoshnirmalya) [inline guides/posts/NLP-apis/post.md]: Please fix this slug.

File: guides/posts/image-apis/post.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Top 5 Image Search and Image Recognition APIs"
+description: "Here are Top 5 Image Search and Image Recognition APIs you can use in your next project."
+slug: "top-image-search-image-recognition-apis"
```
Comment (ghoshnirmalya) [inline guides/posts/image-apis/post.md]: Please fix this slug.

**Review by ghoshnirmalya — CHANGES_REQUESTED** [review]

File: guides/posts/AI-apis/post.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+---
```
Comment (ghoshnirmalya) [inline guides/posts/AI-apis/post.md]: The name of the file should be "ai-apis/post.md". The directory name should be in lowercase as well. Please do the same for the other file as well.

Comment (PrathamKumar14) [reply guides/posts/AI-apis/post.md]: Done!

**Review by ghoshnirmalya — APPROVED** [review]

**Pull request merged**